CacheManager.py

```diff
@@ -0,0 +1,290 @@
+import sqlite3
+import tempfile
+import os
+import urllib.request
+
+'''
+    This class manages local cache
+'''
+
+class CacheManager():
+    def __init__(self):
+        self.tempDir = tempfile.gettempdir()
+        self.dbName = "amigocloud_local_db.db"
+        self.tempPath = os.path.join(self.tempDir, self.dbName)
+        self.isDev = False
+
+    def devPrint(self,content):
+        if self.isDev == True:
+            print(content)
+
+    def openConn(self):
+        conn = sqlite3.connect(self.tempPath)
```

Comment:
Perhaps conn should be a member of a CacheManager  class. It would be easier to handle the connection, and close it when appropriate. Because not it doesn't get closed at all.